src/meta/src/model_v2/actor.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright 2023 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use sea_orm::entity::prelude::*;
+
+use crate::model_v2::I32Array;
+
+#[derive(Clone, Debug, PartialEq, DeriveEntityModel, Eq)]
+#[sea_orm(table_name = "actor")]
+pub struct Model {
+    #[sea_orm(primary_key)]
+    pub actor_id: i32,
+    pub fragment_id: i32,
+    pub status: Option<String>,
+    pub splits: Option<Json>,
+    pub parallel_unit_id: i32,
+    pub upstream_actor_ids: Option<I32Array>,
+    pub dispatchers: Option<Json>,
+    pub vnode_bitmap: Option<String>,
```

Comment:
Why is it a string? Note that only valid UTF-8 is accepted in Rust.

Reply:
I think the implementation of current bitmap if a valid UTF-8 and I was plan to convert it to string manually.

Reply:
> I think the implementation of current bitmap is a valid UTF-8

I'm afraid it's not. 🥵
<img width="597" alt="image" src="https://github.com/risingwavelabs/risingwave/assets/25862682/a2370180-1ef4-4516-8abd-920b4f2bf05f">

Reply:
I will try to store it as a boolean array at implementation time.